axelor-account/src/main/java/com/axelor/apps/account/service/move/PaymentMoveLineDistributionServiceImpl.java

```diff
@@ -80,6 +80,9 @@ public void generatePaymentMoveLineDistributionList(Move move, Reconcile reconci
           new PaymentMoveLineDistribution(
               move.getPartner(), reconcile, moveLine, move, moveLine.getTaxLine());
 
+      if (moveLine.getAccount().getServiceType().getN4dsCode() == null) {
```

Comment:
I can save an account without a service type so maybe there is a potential NPE here